### ISSUE TITLE
Add Observation.component data for all Observations

### ISSUE THREAD

**Issue by StephThaoOng** [hl7au/au-fhir-test-data#68]
Add Observation.component data for all Observations so that there is coverage of all Must Support elements in the test data set. Use [this document](https://csiroau.sharepoint.com/:x:/r/sites/DoHACCSIROdigitalhealthcollaborationspace/_layouts/15/Doc.aspx?sourcedoc=%7B517BA12E-E7FB-4632-93D3-2319A24E7530%7D&file=AU%20Core%20Test%20Report%20-%20%5BProduct%20Name%5D%20-%20Profiles.xlsx&action=default&mobileredirect=true) as a reference to what data is not currently included in the test data set.
In summary, the following profiles are targeted:
- bodyheight, 
- bodytemp, 
- bodyweight, heartrate, 
- resprate, 
- smokingstatus, 
- waistcircum
- diagnosticresult is covered in issues#66

**Comment by heathfrankel** [comment]
The latest list from Danielle in [AU Core FHIR IG Testing](https://confluence.hl7.org/display/HAFWG/AU+Core+FHIR+IG+Testing#AUCoreFHIRIGTesting-AdditionalAUCoreTestingDesired) is:

- Body Height (measurement device: 1259000006 [valueset]( https://build.fhir.org/ig/HL7/cimi-vital-signs/ValueSet-heightLengthMeasDeviceVS.html))
- Body Temperature (measurement device: 8326-1 [valueset](https://build.fhir.org/ig/HL7/cimi-vital-signs/ValueSet-bodyTempMeasurementDeviceVS.html))
- Body Weight (clothing component: 8352-7 [valueset](https://build.fhir.org/ig/HL7/cimi-vital-signs/ValueSet-BodyWeightAssociatedSituationVS.html))
- Heart Rate (exertion phase: 128976003 [valueset](https://build.fhir.org/ig/HL7/cimi-vital-signs/ValueSet-exertionPhaseVS.html) or rhythm component: 301112006 [valueset](https://fhir.test.medirecords.com/ValueSet-mr-heart-rate-rhythm-code.html))
- Respiration Rate (exertion phase: 128976003 [valueset](https://build.fhir.org/ig/HL7/cimi-vital-signs/ValueSet-exertionPhaseVS.html) or body position component)
- Waist Circumference (measurement device: 1259000006 [valueset]( https://build.fhir.org/ig/HL7/cimi-vital-signs/ValueSet-heightLengthMeasDeviceVS.html))

**Comment by heathfrankel** [comment]
Suggested measurement state components:

Body Height (measurement device: 1259000006 done)
Body Temperature (measurement device: 8326-1 done)
Body Weight (clothing: 8352-7 done)
Heart Rate (rhythm: 8884-9 done)
Respiration Rate (exertion phase: 74008-4 done)
Waist Circumference (measurement device: )
